input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,506 @@
 
-The table below provides a high-level comparison, indicating if a resource conformant to a particular AU Core profile also satisfies (i.e. is conformant to) a profile from [International Patient Access](https://hl7.org/fhir/uv/ipa/STU1/), [International Patient Summary](https://hl7.org/fhir/uv/ips/STU1.1/), or [US Core](https://hl7.org/fhir/us/core/STU6.1/). See [Relationship to AUCDI and other IGs](https://build.fhir.org/ig/hl7au/au-fhir-core/branches/ft_blockvote-1/relationship.html#relationship-to-aucdi-and-other-igsl) for more information on the relationship of AU Core to these implementation guides.
+The comparison highlights where AU Core aligns with these national and international guides and identifies additional requirements necessary to meet their standards. 
 
-Conformance in reverse is not guaranteed, i.e. a resource conforming to [International Patient Access](https://hl7.org/fhir/uv/ipa/STU1/), [International Patient Summary](https://hl7.org/fhir/uv/ips/STU1.1/), or [US Core](https://hl7.org/fhir/us/core/STU6.1/) **MAY NOT** conform to AU Core.
+For more information on how AU Core relates to other implementation guides, see [Relationship with other IGs](relationship.html).
 
-<table border="1" cellspacing="0" cellpadding="0">
+### Profile comparison
+The profile comparison highlights how AU Core profiles align with other key FHIR implementation guides, such as International Patient Access (IPA), International Patient Summary (IPS), and US Core.  
+
+A resource can comply with multiple implementation guides, such as AU Core, other national implementation guides, or international implementation guides.Compliance requires meeting specific requirements and constraints, including mandatory elements, cardinality, data types, terminology bindings, and extensions.
+
+The comparison table shows where a resource compliant with AU Core also meets the requirements of profiles in International Patient Access, International Patient Summary, or US Core. Note that compliance in the reverse direction is not guaranteed, i.e. a resources compliant with International Patient Access, International Patient Summary, or US Core **MAY NOT** be compliant with AU Core. Future updates may include reverse comparisons and CapabilityStatement evaluations.
+
+**Legend:**
+
+<img src="green_checkmark.svg.png" width="20"/> **Compliant**: An AU Core compliant resource meets all requirements of the compared profile.
+
+<img src="orange_checkmark.svg.png" width="20"/> **Additional requirements**: An AU Core compliant resource is compatible, but additional changes may be needed to meet all requirements of the compared profile. Where additional requirements are identified, more information is provided in the sections below.
+
+<img src="cross_red_circle.svg.png" width="20"/> **Incompatible**: An AU Core compliant resource is incompatible with the compared profile. A resource cannot be compliant to both.
```

Comment:
Add to end of incompatible: Where incompatible requirements are identified, more information is provided in the sections below.

Reply:
Done